styles.css

```diff
@@ -147,5 +446,154 @@ section {
     background-position: 100% 100%;
   }
 
- 
+  .container h1 {
+    font-family: 'Poppins' "Trebuchet MS", "Lucida Sans Unicode", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
```

Comment:
- [ ] As mentioned by the previous review :point_up: kindly stick to the Figma design your `h1` tag is not the same as the Figma design and also `p` tag is not the same as the Figma design kindly fix it check below :point_down: the comparison between your design and Figma design :+1: 

**Your Design**
![Screenshot from 2023-04-07 02-31-00](https://user-images.githubusercontent.com/45226848/230500456-0b5d9357-4315-4144-b086-7102bdf00e14.png)

**Figma Design**
![Figma design](https://user-images.githubusercontent.com/102699958/230347534-bf8f1414-82ed-45d5-b667-d7be544c19e4.png)